src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -303,12 +365,15 @@ private void AddPythonEnginesOptions()
         /// <summary>
         /// The PreferencesViewModel constructor basically initialize all the ItemsSource for the corresponding ComboBox in the View (PreferencesView.xaml)
         /// </summary>
-        public PreferencesViewModel()
+        public PreferencesViewModel(PreferenceSettings preferenceSettings, DynamoPythonScriptEditorTextOptions editTextOptions)
```

Comment:
You'll need to add another constructor and leave this default one, you can mark it obsolete if you think it should not longer be used.

Reply:
@mjkkirschnner `PreferencesViewModel` is only introduced in 2.12.0, I thought we could touch it this way?

Reply:
sorry guys! You're right, hard for me to keep track!

Reply:
Good, will update on the next commit.
Just out of curiosity, given that the constructor with no parameters will make the class obsolete and probably even crash if trying to change the properties that make us of things like the PreferenceSettings, why do we need to have a constructor with no parameters?

Reply:
@Astul-Betizagasti It only make sense to exit for a short while when we have to be API compatible, but in this case we can touch it.

Reply:
Oh, my browser hadn't updated and I posted that message only seeing Michael's first comment. I will leave it as it was then. Thanks for the review and comments guys!